.github/workflows/slack-prerelease-branch-notification.yml

```diff
@@ -0,0 +1,67 @@
+name: Slack prerelease branch notification
+on:
+  create:
+    branches:
+      - 'prerelease'
+  delete:
+    branches:
+      - 'prerelease'
```

Comment:
I'm not sure if I'm testing this correctly, but do branch filters for the create/delete events work like this? This action seems to run regardless of the new branch name I create for example 🤔

Reply:
You're right, GH doesn't let you filter these. Grr. We can still filter it at the job level, but it's annoying it'll have to run on every branch creation.